sdk/core/azure-core/test/ut/operation_test.hpp

```diff
@@ -60,7 +58,14 @@ namespace Azure { namespace Core { namespace Test {
       return Response<std::string>(m_value, std::make_unique<Http::RawResponse>(*m_rawResponse));
     }
 
+    StringOperation(StringClient const&, std::string const& resumeToken)
```

Comment:
client param first, then resume token? I thought we wanted the other way around. Also, what about context?

Reply:
There's no I/O calls on the unit tests for Operation.  It's a weird case invented by @RickWinter  hahahha